traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -414,6 +414,16 @@ var TableServersController = function(tableName, servers, filter, $scope, $state
 			} catch (e) {
 				console.error("Failure to load stored page size:", e);
 			}
+			
+			try {
+				const page = localStorage.getItem(tableName + "_table_page");
+				const pageSize = $scope.gridOptions.api.paginationGetPageSize();
+				if (page !== undefined && page > 0 && page <= pageSize-1) {
```

Comment:
i think the `page <= pageSize-1` part is breaking it if the pageSize is too low. I set pageSize to 2 rows and then went to page 5 which means it wont go to that page again since it fails that part.  maybe use `$scope.gridOptions.api.paginationGetTotalPages()` instead?

Reply:
i think we will also have to do `$scope.gridOptions.api.paginationSetPageSize(pageSize);` inside this if after the page change

Reply:
Woops, I changed it `pagenationGetTotalPages` but I must've undid that change at some point.

Reply:
Page size shouldn't need to be set as it was set before and it specifies the `max` number of rows to display.